src/Pact/Typechecker.hs

```diff
@@ -809,15 +826,17 @@ assocStepYieldReturns (TopFun (FDefun _ _ _ Defpact _ _ _) _) steps =
         -- yield in previous step, no resume in current step
         (Just{},_,Nothing) -> die' (_aId n) "Missing resume after yield"
         -- resume in current step, no yield in last step
-        (Nothing,_,Just {}) -> die' (_aId n) "Missing yield before resume"
+        (Nothing,_,Just{}) -> die' (_aId n) "Missing yield before resume"
```

Comment:
A number of whitespace changes in this PR are opinionated and different than the rest of the module (ie deleting space here, adding space inconsistently to tuples elsewhere), and slows reviews. Please keep them to a minimum when modifying code.

Reply:
I've reverted everything that isn't PR related